Organic/AdsTxt.php

```diff
@@ -22,24 +23,18 @@ public function __construct( Organic $organic ) {
     }
 
     public function get() {
-         return $this->organic->getOption( 'organic::ads_txt' );
+        return sprintf( self::ADS_TXT_URL_TEMPLATE, $this->organic->getSiteId() );
     }
 
     public function show() {
         if ( isset( $_SERVER ) && $_SERVER['REQUEST_URI'] === '/ads.txt' ) {
             $enabled = $this->organic->getOption( 'organic::enabled' );
 
             if ( $enabled ) {
-                $adsTxt = $this->organic->getOption( 'organic::ads_txt' );
-                header( 'content-type: text/plain; charset=UTF-8' );
-                header( 'cache-control: public, max-age=86400' );
-                echo $adsTxt;
+                header( 'Location: ' . $this->get() );
```

Comment:
Oh! And probably we should add a comment that only one redirect is allowed for `/ads.txt`
Per Ads.txt specification:
> Only a single HTTP redirect to a destination outside the original root domain is allowed to facilitate one-hop delegation of authority to a third party's web server domain.

It's not an issue in this case but comment could help prevent problems in future when someone else will work on this

Reply:
I'll add a comment.